x-pack/plugins/ml/public/application/jobs/new_job/recognize/components/kibana_objects.tsx

```diff
@@ -55,6 +55,11 @@ export const KibanaObjects: FC<KibanaObjectItemProps> = memo(
                       <EuiText size="s" color={exists ? 'subdued' : 'secondary'}>
                         {title}
                       </EuiText>
+                      {success === false && error !== undefined && (
+                        <EuiText size="xs" color="danger">
+                          {error.message}
```

Comment:
Might be worth using `extractErrorMessage` util function in common, but I'm guessing we know for sure the `errorObj` will always have the `message` property in this case.

Reply:
this error isn't in a format that is recognised by `extractErrorMessage`